content/content-strategy/creating-user-centred-content/content-design/top-tasks/case.md

```diff
@@ -0,0 +1,16 @@
+---
+layout: cards/cards
+cards:
+- headline: Topic analysis and top task links
+  text: Read how GOV.UK has put top tasks in action.
+  link: 'https://insidegovuk.blog.gov.uk/2013/11/29/topic-analysis-and-top-tasks/'
+  cta: Read on GOV.UK blog
+- headline: Top tasks management for UK councils
+  text: 
```

Comment:
No text here? intentional?

Reply:
At this stage yes, will need to supply copy

Reply:
ok